composer.json

```diff
@@ -23,9 +23,9 @@
     "require": {
         "php": "^7.3 || ~8.0.0 || ~8.1.0",
         "laminas/laminas-config": "^3.7",
-        "laminas/laminas-crypt": "^3.8",
+        "laminas/laminas-crypt": "~3.6.0",
         "laminas/laminas-http": "^2.15",
-        "laminas/laminas-i18n": "^2.16",
+        "laminas/laminas-i18n": "~2.13.0",
```

Comment:
I think these should stay as they are.

What we could do is setting `config.platform.php: "7.3.99"`, so it knows it needs to do a `composer update` with PHP 7.3 compatibility.

Dropping PHP 7.3 may be necessary too (bumping to 7.4)